tests/dags/test_example_bash_operator.py

```diff
@@ -16,13 +16,12 @@
 # specific language governing permissions and limitations
 # under the License.
 from datetime import timedelta
-
+import datetime
```

Comment:
If `from datetime import timedelta` already present why not doing
`from datetime import datetime, timedelta`

and use `datetime(2022,1,1)` rather than `datetime.datetime(2022,1,1)`

Reply:
oh! didn't see that there. There are a lot of files and I checked each one manually for maximum efficiency so forgot to use some of the already present libraries. If you want I can change them again quickly